secboot/encrypt_dummy.go

```diff
@@ -0,0 +1,34 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
should this be renamed to "fake", to make the inclusive name checker happy?

Reply:
we should rename them but we have more of these *_dummy.go already, so I would like to do that in a follow-up